sphinx/texinputs/sphinxmanual.cls

```diff
@@ -86,38 +87,54 @@
 
 % Catch the end of the {abstract} environment, but here make sure the abstract
 % is followed by a blank page if the 'openright' option is used.
-%
+
+% Note added (Sphinx 1.4.4): the earlier code was wrong and in fact not really
```

Comment:
The comments should describe behavior or purpose of macros. If you want to note the reason of changes, please note it in commit log.